espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/InvocationOperation.java

```diff
@@ -0,0 +1,240 @@
+                case 6:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5));
+                case 7:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6));
+                case 8:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7));
+                case 9:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8));
+                case 10:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9));
+                case 11:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9), arguments.get(10));
+                case 12:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9), arguments.get(10), arguments.get(11));
+                case 13:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9), arguments.get(10), arguments.get(11), arguments.get(12));
+                case 14:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9), arguments.get(10), arguments.get(11), arguments.get(12), arguments.get(13));
+                case 15:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9), arguments.get(10), arguments.get(11), arguments.get(12), arguments.get(13), arguments.get(14));
+                case 16:
+                    return method.invoke(o, arguments.get(0), arguments.get(1), arguments.get(2), arguments.get(3), arguments.get(4), arguments.get(5), arguments.get(6), arguments.get(7), arguments.get(8), arguments.get(9), arguments.get(10), arguments.get(11), arguments.get(12), arguments.get(13), arguments.get(14), arguments.get(15));
```

Comment:
these several lines are too long. Please try to limit line length at 100-120 chars

Reply:
fixed